Project.toml

```diff
@@ -21,6 +22,7 @@ ArraysOfArrays = "0.5, 0.6"
 ElasticArrays = "1"
 EncodedArrays = "0.4"
 HDF5 = "0.15, 0.16, 0.17"
+LegendDataManagement = "0.2, 0.3"
```

Comment:
There is no LegendDataManagement 0.3 yet (?)
https://github.com/legend-exp/LegendDataManagement.jl/releases